projects/config/amqp_setup.py

```diff
@@ -0,0 +1,105 @@
+import pika
+
+from dotenv import load_dotenv
+from os import getenv
+
+load_dotenv()
+RABBITMQ_HOSTNAME = getenv('RABBITMQ_HOSTNAME')
+RABBITMQ_USERNAME = getenv('RABBITMQ_USERNAME')
+RABBITMQ_PASSWORD = getenv('RABBITMQ_PASSWORD')
+RABBITMQ_PORT = getenv('RABBITMQ_PORT')
+
+ROUTING_KEY = 'routing_key'
+BINDING_KEY = 'binding_key'
+PROJECTS_CREATED_QUEUE = 'projects_created'
+PROJECTS_VERIFIED_QUEUE = 'projects_verified'
+PROJECTS_MILESTONES_ADD_QUEUE = 'projects_milestone_added'
+PROJECTS_MILESTONES_OFFSETS_RESERVE_QUEUE = 'projects_milestones_offsets_reserve'
+PROJECTS_MILESTONES_OFFSETS_COMMIT_QUEUE = 'projects_milestones_offsets_commit'
+PROJECTS_MILESTONES_OFFSETS_ROLLBACK_QUEUE = 'projects_milestones_offsets_rollback'
+PROJECTS_MILESTONES_REWARD_QUEUE = 'projects_milestones_reward'
+PROJECTS_MILESTONES_PENALISE_QUEUE = 'projects_milestones_penalise'
+
+
+QUEUES = { # {queue_name: {"binding_key": binding_key, "routing_key": routing_key}, ...}
```

Comment:
just realised this, the queue shouldn't be declared nor bound at the project MS, instead, it should only be publishing to the exchange with the defined routing key.

the responsibility to create queues and binding those queues to the topic exchange should be borne by the consumer, not the producer